synapse/tests/test_eventbus.py

```diff
@@ -7,6 +7,24 @@
 
 from synapse.tests.common import *
 
+@firethread
+def send_sig(pid, sig):
+    time.sleep(0.1)
```

Comment:
hrm, tests with races ( even nice wide ones ) have bitten us previously.  we should probably make the test use a subprocess and send it the signal ( with the handler doing something the test can check for ) to eliminate races.